pkgs/development/python-modules/imgaug/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchPypi, numpy, scipy, scikitimage, opencv3, six, dask }:
+
+buildPythonPackage rec {
+  pname = "imgaug";
+  version = "0.2.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1wy8ydkqq0jrwxwdv04q89n3gwsr9pjaspsbw26ipg5a5lnhb9c2";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    scikitimage
+    opencv3
+    six
+    dask
+  ];
+
+  doCheck = false;
```

Comment:
Include the motivation for disabling tests as a comment.

Reply:
This is due to segmentation fault. And there being no tests?

Reply:
Yes, but you can't know that when looking at the expression.